geode-core/src/test/java/com/gemstone/gemfire/internal/offheap/OffHeapStoredObjectJUnitTest.java

```diff
@@ -42,6 +42,10 @@
 @Category(UnitTest.class)
 public class OffHeapStoredObjectJUnitTest extends AbstractStoredObjectTestBase {
 
+  static {
```

Comment:
Remove this static block. I talked with Kirk and we think that java assertions are enabled consistently from both gradle and the IDE